resources/views/includes/station-autocomplete.blade.php

```diff
@@ -12,29 +12,28 @@
                     </div>
                     <form action="{{ route('trains.stationboard') }}" method="get" id="autocomplete-form">
                         <input type="hidden" id="autocomplete-provider" name="provider" value="train">
+                        <div id="station-autocomplete-container">
+                            <div class="input-group mb-2 mr-sm-2">
+                                <input type="text" id="station-autocomplete" name="station" class="form-control"
+                                       placeholder="{{ __('stationboard.station-placeholder') }} / DS100"
+                                       @isset(request()->station) value="{{request()->station}}" @endisset
+                                />
 
-                        <div class="input-group mb-2 mr-sm-2">
-                            <input type="text" id="station-autocomplete" name="station" class="form-control"
-                                   placeholder="{{ __('stationboard.station-placeholder') }} / DS100" @isset(request()->station) value="{{request()->station}}" @endisset
-                            />
+                                @if($latest->count() > 0 || Auth::user()->home)
+                                    <div class="btn btn-outline-dark" data-mdb-toggle="collapse"
+                                         data-mdb-target="#last-stations"
+                                         title="{{__('stationboard.last-stations')}}">
+                                        <i class="fa fa-history"></i>
+                                    </div>
+                                @endif
 
-                            @if($latest->count() > 0 || Auth::user()->home)
-                                <div class="input-group-append" id="history-button"
-                                     title="{{__('stationboard.last-stations')}}">
-                                <span class="input-group-text">
-                                    <i class="fa fa-history"></i>
```

Comment:
![image](https://user-images.githubusercontent.com/4103693/114886092-8b106800-9e07-11eb-9e11-608a27e17254.png)
The history button is not visible.

Reply:
![image](https://user-images.githubusercontent.com/4103693/114886697-0bcf6400-9e08-11eb-9b87-597c5868990c.png)
Nvm, It was my fault, I didn't have checkins yet. I don't like the look of the black buttons though. Can we make them gray?